modules/placement-driver/build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
     integrationTestImplementation(testFixtures(project(':ignite-distribution-zones')))
     integrationTestImplementation(testFixtures(project(':ignite-runner')))
     integrationTestImplementation(testFixtures(project(':ignite-replicator')))
+    integrationTestImplementation(testFixtures(project(':ignite-transactions')))
```

Comment:
Why? Same question for index/build.gradle and compute/build.gradle.

Reply:
It's because of TestWrappers which I moved to transactions from runner. I thought about moving them to core but there are many of higher-level entities like TableImpl.